pkgs/data/misc/tzdata/default.nix

```diff
@@ -42,6 +42,10 @@ stdenv.mkDerivation rec {
     "CFLAGS+=-DHAVE_DIRECT_H"
     "CFLAGS+=-DHAVE_SYMLINK=0"
     "CFLAGS+=-DRESERVE_STD_EXT_IDS"
+  ] ++ lib.optionals stdenv.isDarwin [
+    # Remove after https://github.com/eggert/tz/commit/5db8b3ba4816ccb8f4ffeb84f05b99e87d3b1be6
```

Comment:
Applies that patch cleanly onto 2022f? If so we should just fetch it.

Reply:
Sure, just did that.